src/ui/sass/modules/_Results.scss

```diff
@@ -213,6 +213,7 @@ $results-cards-margin: calc(var(--yxt-base-spacing) / 2) !default;
 
   &-title
   {
+    margin: 0;
```

Comment:
was this necessary to maintain the styling after changing to h2 from div?

Reply:
yup, otherwise user-agent styling gives it a huge margin

Reply:
(which doesn't match the mocks)

Reply:
![image](https://user-images.githubusercontent.com/23005393/84426750-ba57b680-abf1-11ea-8099-fb041d4c591b.png)